### PR TITLE
Add timeout check during org setup

### PR DESCRIPTION
**Pull request by yash-bitpod**
- add timeout error if org setup takes over 1:30 min

### CODE REVIEW

File: pages/get-started/index.vue [hunk 1]
```diff
@@ -298,45 +303,54 @@ export default {
...
-                        )
-                    })
-                    localStorage.clear()
-                    location.href = `https://${this.orgName}-${this.$config.axios.backendBaseUrl}${this.$config.basePublicPath}/apps/event/list/Event/live-and-draft-event`
+                if (
+                  Object.keys(jobInfo).length &&
+                  new Date() - new Date(jobInfo.createdDate) < 90000
+                ) {
+                  if (jobInfo._SetupErrors.length) {
+                    this.snackbarText = 'Failed to setup your organisation.'
```
Comment (SukantGujar) [inline pages/get-started/index.vue:311]: No harcoded strings. Use localization.

**Review by SukantGujar — APPROVED** [review]

File: pages/get-started/index.vue [hunk 2]
```diff
@@ -298,45 +303,54 @@ export default {
...
+                    this.snackbar = true
+                    this.tab = 1
+                    this.processing = false
+                    this.resetBtn = !this.resetBtn
+                  } else if (jobInfo._SetupStatus.length) {
+                    const lastStatus =
+                      jobInfo._SetupStatus[jobInfo._SetupStatus.length - 1]
+                    if (
+                      lastStatus.Message === 'full setup completed' &&
```
Comment (SukantGujar) [inline pages/get-started/index.vue:320]: No hardcoded strings, use localization.

**Review by SukantGujar — CHANGES_REQUESTED** [review]

File: pages/get-started/index.vue (hunk 2 above)
Comment (yash-bitpod) [reply pages/get-started/index.vue:320]: This isn't a user displayed string, its checking the message in the API response for verifying that the setup completed.

File: pages/get-started/index.vue (hunk 1 above)
Comment (yash-bitpod) [reply pages/get-started/index.vue:311]: Switching these!

**Review by SukantGujar — APPROVED** [review]

**Pull request merged**